docs/sdk-and-tools/indices/validators.md

```diff
@@ -0,0 +1,29 @@
+---
+id: validators
+title: validators
+---
+
+The `_id` field for this index is composed in this way: `{shardID_epoch}` (example: `1_123`)
```

Comment:
for -> of

Reply:
`{shardID_epoch}` => `{shardID}_{epoch}`

Reply:
done